test/e2e/cross-account/smoke.sh

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env bash
+
+THIS_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+ROOT_DIR="$THIS_DIR/../../.."
+SCRIPTS_DIR="$ROOT_DIR/scripts"
+
+source "$SCRIPTS_DIR/lib/common.sh"
+source "$SCRIPTS_DIR/lib/k8s.sh"
+source "$SCRIPTS_DIR/lib/testutil.sh"
+
+
+# AWS_ACCOUNT_ID_2 should be configured to allow cross account resources management
+# from AWS_ACCOUNT_ID
+AWS_ACCOUNT_ID_2=""
+DEFAULT_REGION="eu-west-2"
```

Comment:
Note that there is already an `AWS_REGION` environment variable defined in the `kind-build-test.sh` script:

https://github.com/aws/aws-controllers-k8s/blob/b39842d4ce92595478c4e9e6af4a9667b33985e4/scripts/kind-build-test.sh#L20

So, probably best to call this something like `AWS_REGION_ALT`?